lib/pbench/server/database/models/users.py

```diff
@@ -112,14 +125,12 @@ def delete(username):
             raise
 
     def is_admin(self):
-        # TODO: Add notion of Admin user
-        """this method would always return false for now until we add a notion of Admin user/group.
-        Once we know the admin credentials this method can check against those credentials to determine
-        whether the user is privileged to do more.
-
+        """This method checks whether the given user has an admin role.
         This can be extended to groups as well for example a user belonging to certain group has only those
         privileges that are assigned to the group.
         """
+        if self.role == Roles.ADMIN:
+            return True
         return False
```

Comment:
Or how about simply `return self.role == Roles.ADMIN`??

Also, you can add attributes to an ENUM, making admin access an attribute of a role rather than identical to the name of the role, which would allow `return self.role.isAdmin()`... ;-) (e.g., look at my tracker.py `States` enum.)